src/components/layout/index.js

```diff
@@ -8,8 +8,8 @@
 import React from "react"
 import PropTypes from "prop-types"
 import { useStaticQuery, graphql } from "gatsby"
-
-import Header from "../header"
+import Navbar from "../navbar/navbar"
```

Comment:
you can change this to _**components/navbar**_ and then add a index.js file and move all related components into one file and export them, look at components/list for example

Reply:
Will do, thanks